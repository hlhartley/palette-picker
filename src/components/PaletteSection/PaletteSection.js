import React from 'react';
import PaletteControls from '../PaletteControls/PaletteControls';
import ColorBox from '../ColorBox/ColorBox';

export const PaletteSection = () => {
    return (
        <div className="PaletteSection">
            <PaletteControls/>
            <ColorBox/>
        </div>
    )
}

export default PaletteSection;