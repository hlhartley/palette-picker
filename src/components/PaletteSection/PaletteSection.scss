.PaletteSection {
    border: 2px solid blue;
    width: 49%;
}