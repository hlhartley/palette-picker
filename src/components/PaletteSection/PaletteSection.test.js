import React from 'react';
import { shallow } from 'enzyme';
import { PaletteSection } from './PaletteSection';

describe('PaletteSection', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <PaletteSection/>
      )
    });

  describe('PaletteSection compoment', () => {
    it('should properly render the component elements', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});