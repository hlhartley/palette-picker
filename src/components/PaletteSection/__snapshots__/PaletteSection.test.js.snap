// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaletteSection PaletteSection compoment should properly render the component elements 1`] = `
<div
  className="PaletteSection"
>
  <Connect(PaletteControls) />
  <Connect(ColorBox) />
</div>
`;
